drivers/display/display_sdl.c

```diff
@@ -428,6 +428,43 @@ static int sdl_display_read(const struct device *dev, const uint16_t x,
 	return 0;
 }
 
+static int sdl_display_clear(const struct device *dev)
+{
+	size_t size = 0;
```

Comment:
This is not blocking, but the The "reverse Christmas tree" notation is more popular.

```
	const struct sdl_display_config *config = dev->config;
	struct sdl_display_data *disp_data = dev->data;
	uint8_t bgcolor = 0x00u;
	size_t size = 0;
```